### PR TITLE
Introduce an OpenSSF Scorecard profile

### PR DESCRIPTION
**Pull request by ethomson**
One of _my_ goals using Minder is to improve my repository's OpenSSF Scorecard score, since Minder can automatically remediate findings. Minder does not yet have rules to evaluate the full scope of the OpenSSF Scorecard, nor remediate it, but it will still help detect and improve findings.

### CODE REVIEW

File: profiles/github/openssf_scorecard.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - type: security_policy              # Security-Policy
+    def:
+      filename: SECURITY.md
+  - type: default_workflow_permissions # Token-Permissions
+artifact:
+  # Signed-Releases
+  - type: artifact_signature
+    params:
+      tags: [main]
+      name: test
```
Comment (jhrozek) [inline profiles/github/openssf_scorecard.yaml:61]: I think we can omit the `name` and then the ruletype will apply to all artifacts

File: profiles/github/openssf_scorecard.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - type: license                      # License
+    def:
+      license_filename: LICENSE
+      license_type: ""
+  # Maintained
+  - type: actions_check_pinned_tags    # Pinned-Dependencies for Actions
+  # Packaging
+  - type: codeql_enabled               # SAST
+    def:
+      languages: [go, javascript, typescript]
```
Comment (jhrozek) [inline profiles/github/openssf_scorecard.yaml:50]: I /think/ we can omit the languages and rely on codeQL's autodetection and autobuild (but I will verify this before approving)

File: profiles/github/openssf_scorecard.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      package_ecosystem: pypi
+      schedule_interval: ""
+      apply_if_file: requirements.txt
+  # Fuzzing
+  - type: license                      # License
+    def:
+      license_filename: LICENSE
+      license_type: ""
+  # Maintained
+  - type: actions_check_pinned_tags    # Pinned-Dependencies for Actions
```
Comment (ethomson) [inline profiles/github/openssf_scorecard.yaml:46]: Needs a `def`

File: profiles/github/openssf_scorecard.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  # Packaging
+  - type: codeql_enabled               # SAST
+    def:
+      languages: [go, javascript, typescript]
+      schedule_interval: '30 4-6 * * *'
+  - type: security_policy              # Security-Policy
+    def:
+      filename: SECURITY.md
+  - type: default_workflow_permissions # Token-Permissions
```
Comment (ethomson) [inline profiles/github/openssf_scorecard.yaml:55]: Needs a `def`

**Comment by jhrozek** [comment]
@ethomson I took the liberty to push to your branch. The profile now works well and I added the pull_request check.

**Comment by ethomson** [comment]
Thanks @jhrozek - appreciate the help!

**Review by jhrozek — APPROVED** [review]

**Review by blkt — APPROVED** [review]

**Review by JAORMX — APPROVED** [review]

**Pull request merged**